Makefile

```
@@ -11,11 +11,28 @@ GOLANGCI_LINT_VERSION ?= v2.11.3
 	go install github.com/onsi/ginkgo/v2/ginkgo
 
-test: ginkgo
-	ginkgo -r   --succinct --skip-package=tests/e2e,tests/e2e-blobs,bench --label-filter "!e2e"
-
-test-concurrent: ginkgo
-	ginkgo -r -v --nodes=4 --skip-package=bench --label-filter "!e2e"
+.PHONY: gavel
+gavel:
+	@command -v gavel >/dev/null || go install github.com/flanksource/gavel/cmd/gavel@latest
+
+test: gavel
```

checkmake (0.3.2)

[warning] Target body for "test" exceeds allowed length of 5 lines (7).

(maxbodylength)